example/google-sheets/westeros-houses/inc/queries/class-get-westeros-houses-query.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace RemoteDataBlocks\Example\GoogleSheets\WesterosHouses;
+
+use RemoteDataBlocks\Config\QueryContext;
+
+class GetWesterosHousesQuery extends QueryContext {
+	const COLUMNS = [
+		'House',
+		'Seat',
+		'Region',
+		'Words',
+		'Sigil',
+	];
+
+	public array $input_variables = [
+		'row_id' => [
+			'name'      => 'Row ID',
+			'overrides' => [
+				[
+					'target' => 'utm_content',
+					'type'   => 'query_var',
+				],
+			],
+			'type'      => 'id',
+		],
+	];
+
+	public array $output_variables = [
+		'is_collection' => false,
+		'mappings'      => [
+			'rowId'     => [
```

Comment:
maybe should be snake_case for consistency

Reply:
fixed in 7f9547a